bip-0112.mediawiki

```diff
@@ -29,7 +29,7 @@ When executed, if any of the following conditions are true, the script interpret
 * the stack is empty; or
 * the top item on the stack is less than 0; or
 * the top item on the stack has the disable flag (1 << 31) unset; and
-** the transaction version is less than 2; or
+** the transaction version is non-negative and less than 2, i.e. 0 or 1; or
```

Comment:
```suggestion
** the transaction version is less than 2 when interpreted as an unsigned little-endian integer; or
```

Reply:
leaving as-is unless people feel strongly